src/WatchListPage.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+import { getWatchList, logout } from './services/supabase-utils';
+import MovieItem from './MovieItem';
+
+export default function WatchListPage() {
+  const [watchList, setWatchList] = useState([]);
+
+  async function refreshWatchList() {
+    const myWatchList = await getWatchList();
+
+    setWatchList(myWatchList);
+  }
+
+  useEffect(() => {
+    async function load() {
+      refreshWatchList();
```

Comment:
I think since `refreshWatchList()` is already async and you aren't having to await it in your load function you can probably just get rid of `load()` call the function in the `useEffect` like
```
useEffect(() => {
    refreshWatchList();
}, []);
```